setup.cfg

```diff
@@ -34,3 +34,8 @@ follow_imports = silent
 max-line-length = 88
 ignore = E203,E501
 
+[pydocstyle]
+# D107 Missing docstring in __init__ (reason: documented in class docstring)
```

Comment:
usually parameters in init can or cannot make it to instance variables so I would not add this general rule here

Reply:
These are the warnings to be disabled, so listing it here allow us to also place the docstring in the class definition (where Sphinx expects it to be). This is the format used in craft-parts/providers.

Reply:
Right, my comment is wrt something like this construct:

```python
class Foo:
    """Foo class.

    :ivar foobar: a modified bar.
    """

    def __init__(self, bar):
        """Initialize foo with bar.

        :param bar: a number.
        """
        self.foobar = bar * 2
```

But now I see that Class can have `param`, I keep missing that.